scrc/preprocessors/extractors/spider_specific/section_splitting_functions.py

```diff
@@ -25,13 +27,478 @@ def XX_SPIDER(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optio
     pass
 
 
+def VD_Omni(decision: Union[bs4.BeautifulSoup, str], namespace: dict) -> Optional[Dict[Section, List[str]]]:
+    # print("\nVD_Omni\n")
+
+    def get_paragraphs(soup):
+        """
+        Get composition of the decision
+        :param soup:
+        :return:
+        """
+        # by some bruteforce I have found out that the html files in VD_Omni all have a single div with one of the
+        # following classes:
+        possible_main_div_classes = ["WordSection1", "Section1"]
+        # Iterate over this list and find out the current decision is using which class type
+        for main_div_class in possible_main_div_classes:
+
+            div = soup.find_all("div", class_=main_div_class)
+            if (len(div) != 0):
+                class_found = main_div_class
+                break
+        # We expect to have only one main div
+        assert len(div) == 1
+        # If the divs is empty raise an error
+        if (len(div) == 0):
+            message = f"The main div has an unseen class type"
+            raise ValueError(message)
+        paragraphs = []
+        # paragraph = None
+        for element in div:
+            if isinstance(element, bs4.element.Tag):
+                text = str(element.string)
+                # This is a hack to also get tags which contain other tags such as links to BGEs
+                if text.strip() == 'None':
+                    text = element.get_text()
+                paragraph = clean_text(text)
+                if paragraph not in ['', ' ', None]:  # discard empty paragraphs
+                    paragraphs.append(paragraph)
+        return paragraphs
```

Comment:
Make sure to assign sections to the paragraphs. This section splitting can then be used for other tasks as well.

Reply:
I am not sure, this splits the sections correclty. 
The header contains everything before the facts. You can use strings like "Vu les faits suivants" to split. Something like "Considérant en droit" can then be used to split the facts from the considerations.
Does this make sense?